### ISSUE TITLE
UnboundLocalError raised when performing column bar plot

### ISSUE THREAD

**Issue by tournami** [ResidentMario/missingno#21]
Hi ResidentMario,

Thank you for the awesome library.  I'm curious why scikit-learn or pandas haven't created something like this???

My bar column plot will show up but there is an error prior:

`---------------------------------------------------------------------------
UnboundLocalError                         Traceback (most recent call last)
<ipython-input-37-3dee6435d23f> in <module>()
----> 1 msno.bar(train.sample(10))

/Users/Mike/anaconda/envs/py2/lib/python2.7/site-packages/missingno/missingno.pyc in bar(df, figsize, fontsize, labels, log, color, inline, filter, n, p, sort)
    367     # Create the third axis, which displays columnar totals above the rest of the plot.
    368     ax3 = ax1.twiny()
--> 369     ax3.set_xticks(pos)
    370     ax3.set_xlim(ax1.get_xlim())
    371     ax3.set_xticklabels(nullity_counts.values, fontsize=fontsize, rotation=45, ha='left')

UnboundLocalError: local variable 'pos' referenced before assignment`

 It's not a show stopper by any means...

**Comment by ResidentMario** [comment]
Can you provide me with what the data you have in `train` is? I see where this error stems from, but I need a test set with which to implement the fix.

**Comment by tournami** [comment]
Sure it's on Kaggle:

[train2016.csv](https://inclass.kaggle.com/c/can-we-predict-voting-outcomes/download/train2016.csv)

I placed it in a pandas data frame:

```
import pandas as pd
import missingno as msno
%matplotlib inline
msno.bar(pd.read_csv("train2016.csv", index_col="USER_ID"))
```

**Comment by ResidentMario** [comment]
Addressed in 0.3.3. Thanks!
